www/js/app.js

```diff
@@ -29,13 +36,24 @@ class App extends React.Component {
   }
 }
 
+const local = new PouchDB('stop-here-db');
+const gateway = new Gateway(local);
+
+const storeBuilder = new StoreBuilder([ gateway.getMiddleware() ]);
+const store = storeBuilder.build();
+
```

Comment:
The builder allows us to create a store with additional dependencies (the middleware) that could not be required at module load time